.github/CODEOWNERS

```diff
@@ -1,6 +1,9 @@
 # This file describes the maintainers for the main components. See
 # `dev/doc/MERGING.md`.
 
+# default
+/*      @coq/pushers
```

Comment:
Do keep this change?

Reply:
No!

Reply:
Maybe not, it seems to have strange interactions with individual owners.

Reply:
In fact, you can put me back for extraction, then we clarify the distinction between maintainers and owners. Sorry for the noise.

Reply:
My main concern is not to spam code owners who are not watching the repository and thus should only receive e-mails / notifications when their part of the code is affected.